src/Raven.Server/Documents/StreamsTempFile.cs

```diff
@@ -216,7 +237,7 @@ public override void Write(byte[] buffer, int offset, int count)
 
             public override long Position
             {
-                get => _file.Position + _startPosition;
+                get => _file.Position - _startPosition;
```

Comment:
this looks like a bug fix? do we need to apply this in prev releases?

Reply:
Yep, it is. but we are not using it anywhere (including my changes...)  so I'm not sure